Testes funcionais: Erro no parâmetro de processo aberto para envio de processo em bloco.

**Ambiente:**

 SEI 5.1.0 (release/5.1.0)
 Tramita 4.1.0 (release/4.1.0)
 Assinatura (release/1.4.0)

**Descreva o erro:**

Problemas ocorrem nos testes abaixo:

**Tramite Bloco De Tramite Situação Processo Concluído**
  - Validar situação do processo no bloco status6
  - Validar situação do processo no bloco status9

**Tramite Bloco Externo Com Processo Não Mapeado Recusa**
 - Verificar envio tramite em bloco

**Tramite Processo Bloco De Tramite Permissões**
  - Verificar envio processo

-----


**Comportamento esperado**

O sistema deve permitir a seleção e o envio em lote de múltiplos processos abertos, validando corretamente os parâmetros de cada um. A tramitação deve ser concluída com sucesso para todos os itens selecionados, sem interrupções por erro de parâmetro.

-----

**Screenshots**

Ambiente: SEI 5.1.0 (release/5.1.0)
          Tramita 4.1.0 (release/4.1.0)
          Assinatura (release/1.4.0)

============= ERRO 1

Tramite Bloco De Tramite Situacao Processo Concluido
✘ Validar situacao do processo no bloco status6
   ┐
   ├ Facebook\WebDriver\Exception\TimeoutException:
   │
   │ /tests/vendor/php-webdriver/webdriver/lib/WebDriverWait.php:71
   │ /tests/tests/CenarioBaseTestCase.php:258
   │ /tests/tests/TramiteBlocoDeTramiteSituacaoProcessoConcluidoTest.php:72
   ┴
✔ Validar situacao do processo no bloco status7
✘ Validar situacao do processo no bloco status9
   ┐
   ├ Facebook\WebDriver\Exception\TimeoutException:
   │
   │ /tests/vendor/php-webdriver/webdriver/lib/WebDriverWait.php:71
   │ /tests/tests/CenarioBaseTestCase.php:258
   │ /tests/tests/TramiteBlocoDeTramiteSituacaoProcessoConcluidoTest.php:242
   ┴

===============================================================================

============= ERRO 2

Tramite Bloco Externo Com Processo Nao Mapeado Recusa
✔ Tramite bloco externo com processo nao mapeado
✔ Verificar envio processo
✘ Verificar envio tramite em bloco
   ┐
   ├ Failed asserting that 1 matches expected 0.
   │
   │ /tests/tests/TramiteBlocoExternoComProcessoNaoMapeadoRecusaTest.php:151
   ┴
✔ Incluir processo recusado em novo bloco

FAILURES!
Tests: 4, Assertions: 7, Failures: 1.
Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port).

make: *** [Makefile:247: test-functional] Erro 1


==================================================================

============= ERRO 3

Tramite Processo Bloco De Tramite Permissoes
✔ Tramite contendo documento interno
✔ Trocar ordenacao documento
✔ Criar excluir processos em bloco externo
✔ Tramite bloco externo
✘ Verificar envio processo
   ┐
   ├ Facebook\WebDriver\Exception\TimeoutException:
   │
   │ /tests/vendor/php-webdriver/webdriver/lib/WebDriverWait.php:71
   │ /tests/tests/CenarioBaseTestCase.php:258
   │ /tests/tests/TramiteProcessoBlocoDeTramitePermissoesTest.php:255
   ┴
✔ Verificar possivel exclusao processo bloco

ERRORS!
Tests: 6, Assertions: 67, Errors: 1.
Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port).


=======================================================
 
----

**Definição de Feito (D.O.D. - Definition of Done):**

- [ ] Evidências do trabalho realizado.
- [ ] Executar Phpcs sem falhas.

-----

--- a/src/rn/ProcessarPendenciasRN.php
+++ b/src/rn/ProcessarPendenciasRN.php
@@ -404,6 +404,7 @@ class ProcessarPendenciasRN extends InfraRN
             $objExpedirProcedimentoDTO->setNumIdBloco($objPenBlocoProcedimentoDTO->getNumIdBlocoProcesso());
             $objExpedirProcedimentoDTO->setNumIdAtividade($objPenBlocoProcedimentoDTO->getNumIdAtividade());
             $objExpedirProcedimentoDTO->setNumIdUnidade($objPenBlocoProcedimentoDTO->getNumIdUnidade());
+            $objExpedirProcedimentoDTO->setBolSinMultiplosOrgaos(false);
             $objExpedirProcedimentoDTO->setBolSinEnvioAutoMultiplosOrgaos(false);
 
             $objExpedirProcedimentoRN = new ExpedirProcedimentoRN();

